<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingScreen</class>
 <widget class="QDialog" name="VotingScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnVote">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cast Vote</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCandidate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Candidate:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCandidate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
